docs/content/comparisons/druid-vs-commericial-solutions.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: doc_page
+---
+
+Druid vs Proprietary Commercial Solutions (Vertica/Redshift)
```

Comment:
We don't need to compare against everything that's ever been invented, just things that people commonly ask "how does Druid compare to X"?